ocsp_status.go

```diff
@@ -126,7 +137,12 @@ func main() {
 			}
```

Comment:
I think panic would forget to close the respFile, we may need a way to recover from the panic before exit

Reply:
`ioutil.ReadFile` also closes the file: https://forum.golangbridge.org/t/reading-a-text-file/3534

So if it errors on reading the file, it seems like `ioutil.ReadFile` will do the right thing and close the file, otherwise if it errors later down the line, the file will already be closed